code/modules/uplink/uplink_items.dm

```diff
@@ -1775,6 +1775,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/clothing/glasses/thermal/syndi
 	cost = 3
 
+/datum/uplink_item/device_tools/syndie_armor_kit
+	name = "Syndicate Armor Kit"
+	desc = "Box of armor identical to used by nanotrasen security officers. Modified to protect from some additional threats and given a new red paint job. \
+			Comes with a free turtleneck"
+	item = /obj/item/storage/box/syndie_kit/armor_kit
+	cost = 2
+	illegal_tech = FALSE
```

Comment:
Why does this not provide illegal tech?

Reply:
Looking at few other items i got the impression it was used to prevent the box itself from granting illegal tech but after further investigation it seems i was confused. Fixed - thanks!

Reply:
Illegal tech used to be applied to any item that could be brought in an uplink, rather than any item brought from an uplink so its mostly safe to use now. Usually only is set to false for things like TC and ammo